### DISCUSSION TITLE
version 5 scTransform and then PCA, i get the following message - In PrepDR(object = object, features = features, verbose = verbose) :   The following 149 features requested have not been scaled

### DISCUSSION THREAD

**Discussion by wjbio15** [satijalab/seurat#9127] [Q&A]
Hi Everyone, 

can anyone offer any insight into the above message? After SCT transform and subsuent PCA, I get the following message..

In PrepDR(object = object, features = features, verbose = verbose) :
  The following 149 features requested have not been scaled... (the genes listed), 

is this a version 5 problem with scTransform? anyone lese having the same problem?

**Comment by Bio-magician** [comment]
I have the same problem about it

**Comment by gevro** [comment]
I'm having the same issue - any insight on this?
